lib/Map/Legend.js

```diff
@@ -115,36 +115,24 @@ var Legend = function(props) {
      */
     this.variableNameTop = defaultValue(props.variableNameTop, 17);
 
+    /**
+     * Gets or sets the CSS class that will be applied to the legend when it is displayed.
+     * This is used to ensure that the correct font is used when measuring text for word wrapping.
+     * @type {String}
+     */
+    this.cssClass = defaultValue(props.cssClass, 'legend__legend');
```

Comment:
I think i totally missed the discussion on slack, but why not use `foreignobject` to treat those text as html? 

Also this one does wrapping as well: http://d3plus.org/examples/utilities/a39f0c3fc52804ee859a/